app/styles/routes/_city.scss

```diff
@@ -54,7 +64,9 @@
 
     &__grid {
       display: grid;
-      row-gap: 25px;
+      row-gap: 30px;
+      -webkit-row-gap: 30px;
+      -moz-row-gap: 30px;
```

Comment:
these vendor prefixes should be added by the pre-processor; no need to include them in the source

Reply:
oh okay but you sure they're really included ? because we have a lack of row-gap on ios mobile's devices (safari)

Reply:
ah yeah I'm seeing that. we can target older browsers via another config, but if it's only to support this property, then this makes sense. go ahead!